clients/privacy-center/cypress/e2e/consent.cy.ts

```diff
@@ -140,4 +140,16 @@ describe("Consent settings", () => {
       });
     });
   });
+
+  describe("when globalPrivacyControl is enabled", () => {
```

Comment:
Note that https://github.com/ethyca/fides/pull/2300 still wouldn't give us a way to dynamically change the config used by `fides-consent.js` because the browser script gets its config at build time. I'm considering adding a mechanism for the script to load a config at runtime, maybe by checking for a `window.Fides.config` object.